CHANGES.md

```diff
@@ -135,6 +135,10 @@
   fixed cost in this computation to better handle cases where the fixed cost
   is larger than a single second. To get the old behavior, one can pass
   `target_batch_duration_secs_including_fixed_cost=1` to BatchElements.
+* Dataflow runner enables sibling SDK protocol for Python pipelines using custom containers on Beam 2.46.0 and newer SDKs.
```

Comment:
Shall we update https://cloud.google.com/dataflow/docs/guides/using-custom-containers#use_an_base_image?

Reply:
there is a callout under *Important* in https://cloud.google.com/dataflow/docs/guides/using-custom-containers#use_a_custom_base_image_or_multi-stage_builds . But we could also add this to troubleshooting.